operator_ui/src/connectors/redux/reducers/bridges.ts

```diff
@@ -0,0 +1,49 @@
+export interface IState {
+  items: object
```

Comment:
I think this too should be `{[id: string]: object}`, because in the opposite case, the test cases with `state.bridges.items.a` will throw an error on `.a`